app/api/route/schemas/user_schemas.py

```diff
@@ -52,3 +59,18 @@ class UserSchema(marshmallow.Schema):
     # Output only fields in addition to id field
     created_at = fields.DateTime(dump_only=True)
     updated_at = fields.DateTime(dump_only=True)
+
+
+class CreateUserSchema(UserSchema):
+    @marshmallow.post_load
+    def make_user(self, data: dict, **kwargs: dict) -> users.CreateUserParams:
+        return users.CreateUserParams(**data)
+
+
+class PatchUserSchema(UserSchema):
+    @marshmallow.post_load
+    def make_user(self, data: Mapping[str, Any], **kwargs: dict) -> users.PatchUserParams:
```

Comment:
You can do `dict[str, Any]` as of 3.10

Reply:
that's true, but i think `Mapping` is a bit more general, since i don't really care whether `data` is an actual dict or just a dict-like object that supports the `**` unpacking operator

Reply:
Won't it always be a dict in this case though?

Reply:
yeah i believe so. i can change it back, dict is simpler and we don't need the generalization